### PR TITLE
fixed pids

### PR DESCRIPTION
**Pull request by Techno-Fox**

### CODE REVIEW

File: src/psutil/psutil_linux.nim [hunk 1]
```diff
@@ -166,13 +168,13 @@ proc pids*(): seq[int] =
...
 
-    return mapIt( filterIt( all_files, isnumder(it.path) ), parseInt( it.path ) )
+    return mapIt( filterIt( all_files, isnumber(it.path) ), parseInt( it.path ) )
 
 
 proc pid_exists*( pid: int ): bool =
     ## Check For the existence of a unix pid
 
-    let exists = psutil_posix.pid_exists( pid )
+    let exists = psutilim_posix.pid_exists( pid )
```
Comment (johnscillieri) [inline src/psutil/psutil_linux.nim:177]: Is this a typo?

**Comment by johnscillieri** [comment]
@Techno-Fox Thanks for doing this! I'll merge this in and then submit a pull request upstream to @juancarlospaco 's repository.

File: src/psutil/psutil_linux.nim (hunk 1 above)
Comment (Techno-Fox) [reply src/psutil/psutil_linux.nim:177]: yes. I had them in plenty of files. I had them under different filenames to not get mixed up with old and new. I just went to fix them

**Comment by Techno-Fox** [comment]
Welcome

**Pull request merged**